docs/IDE/index.rst

```diff
@@ -3,18 +3,14 @@
 IDE Support
 ===========
 
-Language features
------------------
+The following supported sphinx needs language features are intergrated into `Esbonio language server <https://github.com/swyddfa/esbonio>`_.
```

Comment:
I think we need some docs for IDE extension installation too.
Otherwise it is not clear what the user shall do.

Can you add a chapter `Installation` and a sub-chapter `VsCode`?
Then 1-2 lines of docs for installing the needed RST-Plugin and get it running for Sphinx-Needs.

Document only the Sphinx-Need-specific part. Everything else shall be just a link to the original extension docs